assembl/static2/js/app/components/common/GlobalErrorBoundary.jsx

```diff
@@ -15,7 +18,13 @@ export default class GlobalErrorBoundary extends React.Component<Props, State> {
 
   componentDidCatch(error: Error, info: { componentStack: string }) {
     this.setState({ error: error });
-    // TODO: log the error to sentry
+    Sentry.withScope((scope) => {
+      Object.keys(info).forEach((key) => {
+        scope.setExtra(key, info[key]);
```

Comment:
we have to filter passwords and emails here ...

Reply:
I don't think we have emails here (and I'm pretty sure we don't have passwords). `info` is basically the stack trace from react / js.

Reply:
et si erreur graphql, nous avons besoin de récupérer les valeurs du POST ou du GET...

Reply:
You'll get user id (that's what you got with raven), coming from the request. Otherwise, no passwords or requests should be a key here. And yes the main thing that we need to capture, actually, for sentry, is the root of the exception.

Reply:
@amensouissi assembl/graphql/view.py:20-24

Reply:
yes, I would like to at least have the name of the query / mutation

Reply:
@ayyazdaniaryan @amensouissi  about `LoggingMiddleware`, it is in fact not used. See lines 98 to 100.